app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Kristina's Task Tracker</h1>
+<%= link_to 'Add task', new_task_path %>
+<ol>
+<% @tasks.each do |task| %>
+  <li>Task Name: <%= link_to task.name, task %></li>
+    <p><%= link_to 'Delete', task, :method => :delete, data: {confirm: 'Are you sure?' } %></p>
+    <p><%= link_to 'Mark Complete', complete_task_path(task), :method => :patch %></p>
```

Comment:
It would make sense to only display 'Mark Complete' if the task is incomplete. Fortunately because the What logic would you need to add to make this the case?